.buildkite/test-pipeline.yaml

```diff
@@ -237,7 +237,7 @@ steps:
   source_file_dependencies:
   - vllm/lora
   - tests/lora
-  command: pytest -v -s lora --shard-id=$$BUILDKITE_PARALLEL_JOB --num-shards=$$BUILDKITE_PARALLEL_JOB_COUNT --ignore lora/test_long_context.py lora/test_chatglm3_tp.py lora/test_llama_tp.py
+  command: pytest -v -s lora --shard-id=$$BUILDKITE_PARALLEL_JOB --num-shards=$$BUILDKITE_PARALLEL_JOB_COUNT --ignore=lora/test_long_context.py --ignore=lora/test_chatglm3_tp.py --ignore=lora/test_llama_tp.py
   parallelism: 4
 
```

Comment:
@DarkLight1337  It look like  https://github.com/vllm-project/vllm/blob/main/.buildkite/test-pipeline.yaml#L366 and https://github.com/vllm-project/vllm/blob/main/.buildkite/test-pipeline.yaml#L384  are in a similar situation, is that right?

Reply:
No, the problem is that we need to pass ignore for each item. The equals sign is optional.